controlplane/kubeadm/controllers/controller.go

```diff
@@ -213,6 +213,7 @@ func patchKubeadmControlPlane(ctx context.Context, patchHelper *patch.Helper, kc
 			controlplanev1.MachinesReadyCondition,
 			controlplanev1.AvailableCondition,
 			controlplanev1.CertificatesAvailableCondition,
+			controlplanev1.MachinesCreatedCondition,
```

Comment:
I'm ok having this in the summary, wdyt @fabriziopandini ?

Reply:
I agree! IMO this should go a the top of the list because it should have priority over other conditions